src/components/Input/Input.tsx

```diff
@@ -67,12 +67,11 @@ export const Input = React.forwardRef<HTMLInputElement, Props>(
     ) => {
         const fallbackId = React.useId();
         const idRef = id || fallbackId;
-        const [hasText, setHasText] = React.useState<boolean>(!!value);
+        const { onChange, style } = rest;
```

Comment:
you don't need it to do here. You can just add these where the props are destructured (line 63 or so)